### PR TITLE
[docs] Add appveyor badge

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin

### CODE REVIEW

**Comment by AndreyBelym** [comment]
There is an option to add Travis mascot to the badge, but the logo must be encoded in base64, so the image url will be huge:
Travis Badge

https://img.shields.io/travis/rust-lang/rust.svg?logo=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAABmJLR0QA%2FwD%2FAP%2BgvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4AkQCxYCeVq%2F1gAABB5JREFUOMt9k19IW3ccxc%2Fv5ppfshQqNrdeSTCK2dS1ZaAZVglhHb2rCCtD7agihc039Wn41JfVlxXGBnsrpTA2h9IhhRUfLMvdw1xFk8xai2KNN9f%2F%2BWc0sWuMmpt899IOt0HP6%2BF8OA%2FnMPxHra2tePToETwez3vt7e2fS5L0sc1mqzQM4%2BXu7u50OBz%2B%2Fs6dO2N4kzo6OgAA169f%2F3ZlZYU2NjZI0zSKRCIUiUQoHA5TKpWioaGhA6%2FXK70RNjg4%2BNP8%2FDxlMhkaGRmhjs5O6u7upvb2dvrq9m2amJigVCpFo6OjeQDliqL8H9LZ2fnZwsICxeNxisfj9ODBA%2FpjcpKuXbtGqqpSIBCgu3fvUjKZJE3T6N69e0EAUBQFwkliV1fXkCiKSKfTCIVCEAQBtXV1AACr1QpZlhGPx5FIJKBpGrxe7%2Ftut%2Fsjv98Pk67r8Pl8lr6%2BvtDVq1drp6en4XQ6MTc3BwDgnGNhcRHb29twOBx48eIFzp07h6WlJbjdbjQ3N3%2BaSqWemgDg4sWL39y8efMTi8WCUCiEo6MjbG1tYW9vDxsbG3jLagXnHJFIBIZhIJfLwWq1QhRF1NfXi7Isd4sAQERGPp9HMBhEWVkZAWAOpxO7qRQZhsHMZjMKhQKdPn0apaWlKCkpYcViEeFwGJxzyuVyjAFAT0%2BPp6ysLLS2tga7JOEgm4XNZqN0Os24xQJXZSUqKiow%2B%2BQJOOfIpNNoaGiAKIq0tbXFDg8PY0xRFBARt0vS%2FvHREW9ra0NVVRWePXuGGzduQBAELC8v4%2F79%2B%2FD5fKivr4MgmKBpGqYeP8ba%2BvpmLBarEwFYGGML9jNn4oyxyt7eXra%2Bvo5fHj7ETCAAgTGoqkomk4k1Njair38Awz%2F%2BgKqqKhQKBdhOnaowm81mAcAlv9%2FvbmlpsdrtdtJ1HZIk4W13DXGzGVNTUxgYGGBOp5MWFxfx4aUPKBaNIpfLYXV1FdXV1aLD4Tg06bquvdpJc01NzbvBYBCrq6sUiUSws7MDQRBw5coVtrS0hOfPn6NYLOLP2VnMzMywZDJJ58%2Bf3xseHv6avR6jx%2BM5JcvyO16vd1aWZVy4cAGbm5v0m6oyo1Cgv16%2BZCBCb28vysvLkc%2FnMT4%2B%2Fp2qqr9yzidMr0HRaPTY5XI1ulyuaH9%2Ff8vy8jJGRkdZidkMAhjnHBaLBYFAAIlEgqLb22xycrKhWCz%2B7Pf7V9jJr12%2BfPlLm812i4hglyRwzkFEAADDMCCKIgCAMQbGGHaSyZmxsbFmRVHwTyNFUaCq6u%2B1tbU%2Bh9NZbTK9shhD7uAA%2BeNjYoLAGGPIZDJIxONPs9lsq67rOV3X8a9GiqJgf3%2B%2FpFyWvzAJQhuATcZYU1NTk%2Fvs2bPw%2B%2F0TsVjs1kE2uxAMhQ5OZv8GNznvA%2FCwUGoAAAAASUVORK5CYII%3D

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ It allows you to easily create, maintain and execute automated web tests across
 
 Tests             | Status
 ----------------- | ----------
-All               | [![Build Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe)
+All               | [![Travis CI Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe) [![AppVeyor status](https://ci.appveyor.com/api/projects/status/ftelkyuiji8lyadf?svg=true)](https://ci.appveyor.com/project/DevExpress/testcafe)
```
Comment (inikulin) [inline README.md]: Just put them in separate rows:
- For Travis: `All Travis tasks (server, client, functional mobile, etc.)`
- For AppVeyor `Functional desktop`

**Comment by AndreyBelym** [comment]
FPR

File: README.md [hunk 2]
```diff
@@ -31,7 +31,8 @@ It allows you to easily create, maintain and execute automated web tests across
 
 Tests             | Status
 ----------------- | ----------
-All               | [![Build Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe)
+All Travis tasks (server, client, functional: mobile, macOS, Edge)  | [![Travis CI Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe)
```
Comment (inikulin) [inline README.md]: `MacOS`

Comment (inikulin) [inline README.md]: Also add `[docs]` to PR title - we don't need full CI and we need to lint README

Comment (AndreyBelym) [inline README.md]: It is really `macOS` now: http://www.apple.com/ru/macos/sierra/

**Comment by inikulin** [comment]
@AndreyBelym FPR?

File: README.md (hunk 2 above)
Comment (inikulin) [inline README.md]: Hmm, ok then

**Pull request merged**